elderly_assistant/hardware/board.py

```
@@ -61,6 +61,106 @@ def get_led():
+    非 M10 环境（无 pinpong）自动降级为「空操作」，不影响主流程与其它平台运行。
+    通过引用计数支持 TTS / 蜂鸣器并发调用：最后一个使用者退出时才重新静音。
+    """
+
+    def __init__(self):
+        self._pin = None
+        self._available = False
+        self._lock = threading.Lock()
+        self._refcount = 0
+        self._initialized = False
+
+    def _ensure_init(self):
```

Ruff (0.16.1)

[warning] Missing return type annotation for private function `_ensure_init`

Add return type annotation: `None`

(ANN202)